### DISCUSSION TITLE
Is it possible to run an older version of Fooocus and prevent it from updating?

### DISCUSSION THREAD

**Discussion by Astr0kreep** [lllyasviel/Fooocus#1553] [Q&A]
Hi There,

I would like to know if it's possible to run an older version of Fooocus and prevent it from auto-updating when starting it up. This newer version is causing me problems when trying to use LoRA's where it just worked on the previous version. This ends up costing me hours just trying to get things to work as expected. I have tried the suggestions people have left in other threads and I know the LoRA's are working because they work in automatic1111 SD. They just don't work, no matter how much I crank the weights etc. Any help with getting an older version installed and pegged at that version would be much appreciated.

**Comment by lllyasviel** [comment]
https://github.com/lllyasviel/Fooocus#download-previous-version

**Reply by Astr0kreep** [reply]
Thank you for your answer and pointing me to the steps to downgrade. 

I worry that the script that kicks off when I run the run.bat file will kick off an update to a newer version. The script it's referencing is entry_with_update. I am no developer so I am unsure if this script will cause Fooocus as a whole to update. If this is not the case, then I'll gladly "patch" my current version with the older files as instructed and call it a day. 

Does that script cause Fooocus to update? Or is that something I would have to do manually?
